discord/ext/commands/bot.py

```diff
@@ -516,7 +559,7 @@ def remove_listener(self, func: CoroFunc, name: str = MISSING) -> None:
             except ValueError:
                 pass
 
-    def listen(self, name: str = MISSING) -> Callable[[CFT], CFT]:
+    def listen(self, name: str = MISSING, /) -> Callable[[CFT], CFT]:
```

Comment:
I think this one is fine as a dual keyword/positional.

Reply:
```suggestion
    def listen(self, name: str = MISSING) -> Callable[[CFT], CFT]:
```